rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -617,10 +616,11 @@ namespace rsx
 		while (true)
 		{
 			// Wait for external pause events
-			if (external_interrupt_lock.load())
+			if (external_interrupt_lock)
```

Comment:
This bit is functionally the same as it was before. What happened here?

Reply:
minor cleanup to match the other pause() and unpause() new code.